app/web/src/components/Workspace/WorkspaceAuditLog.vue

```diff
@@ -363,83 +328,103 @@ watch(size, (size) => {
-      const i = currentFilters.value.changeSetFilter.indexOf(filterId);
-      currentFilters.value.changeSetFilter.splice(i, 1);
-    } else currentFilters.value.changeSetFilter.push(filterId);
+    if (logsStore.filters.changeSetFilter.includes(filterId)) {
+      const i = logsStore.filters.changeSetFilter.indexOf(filterId);
+      logsStore.filters.changeSetFilter.splice(i, 1);
+    } else logsStore.filters.changeSetFilter.push(filterId);
+    table.getColumn(id)?.setFilterValue(logsStore.filters.changeSetFilter);
   } else if (id === "entityName") {
-    if (currentFilters.value.entityNameFilter.includes(filterId)) {
-      const i = currentFilters.value.entityNameFilter.indexOf(filterId);
-      currentFilters.value.entityNameFilter.splice(i, 1);
-    } else currentFilters.value.entityNameFilter.push(filterId);
+    if (logsStore.filters.entityNameFilter.includes(filterId)) {
+      const i = logsStore.filters.entityNameFilter.indexOf(filterId);
+      logsStore.filters.entityNameFilter.splice(i, 1);
+    } else logsStore.filters.entityNameFilter.push(filterId);
+    table.getColumn(id)?.setFilterValue(logsStore.filters.entityNameFilter);
   } else if (id === "entityType") {
-    if (currentFilters.value.entityTypeFilter.includes(filterId)) {
-      const i = currentFilters.value.entityTypeFilter.indexOf(filterId);
-      currentFilters.value.entityTypeFilter.splice(i, 1);
-    } else currentFilters.value.entityTypeFilter.push(filterId);
+    if (logsStore.filters.entityTypeFilter.includes(filterId)) {
+      const i = logsStore.filters.entityTypeFilter.indexOf(filterId);
+      logsStore.filters.entityTypeFilter.splice(i, 1);
+    } else logsStore.filters.entityTypeFilter.push(filterId);
+    table.getColumn(id)?.setFilterValue(logsStore.filters.entityTypeFilter);
   } else if (id === "title") {
-    if (currentFilters.value.titleFilter.includes(filterId)) {
-      const i = currentFilters.value.titleFilter.indexOf(filterId);
-      currentFilters.value.titleFilter.splice(i, 1);
-    } else currentFilters.value.titleFilter.push(filterId);
+    if (logsStore.filters.titleFilter.includes(filterId)) {
+      const i = logsStore.filters.titleFilter.indexOf(filterId);
+      logsStore.filters.titleFilter.splice(i, 1);
+    } else logsStore.filters.titleFilter.push(filterId);
+    table.getColumn(id)?.setFilterValue(logsStore.filters.titleFilter);
   } else if (id === "userName") {
-    if (currentFilters.value.userFilter.includes(filterId)) {
-      const i = currentFilters.value.userFilter.indexOf(filterId);
-      currentFilters.value.userFilter.splice(i, 1);
-    } else currentFilters.value.userFilter.push(filterId);
+    if (logsStore.filters.userFilter.includes(filterId)) {
+      const i = logsStore.filters.userFilter.indexOf(filterId);
+      logsStore.filters.userFilter.splice(i, 1);
+    } else logsStore.filters.userFilter.push(filterId);
+    table.getColumn(id)?.setFilterValue(logsStore.filters.userFilter);
```

Comment:
This isn't the best solution for the long term, but the filters are still in the pinia store so that they are saved for the given change set as the user moves throughout the application. However, the `setFilterValue` call is doing the real work here.